lib/static/new-ui/features/suites/components/SuitesTreeView/utils.ts

```diff
@@ -342,6 +404,31 @@ export const sortTreeNodes = (entitiesContext: EntitiesContext, treeNodes: TreeN
             tags.push(`${failedRunsCount} ${(failedRunsCount === 1 ? 'failed run' : 'failed runs')}`);
         }
 
+        const duration = weight.metadata.duration;
+        if (duration !== undefined) {
+            const durationSeconds = Math.round(duration / 1000 * 10) / 10;
+            let averageDurationSeconds = 0;
+            if (weight.metadata.runsCount && weight.metadata.runsCount > 1) {
+                averageDurationSeconds = Math.round(durationSeconds / weight.metadata.runsCount * 10) / 10;
+            }
+
+            if (durationSeconds < 0.1) {
+                tags.push('<0s in total');
```

Comment:
How it could be "<0s"? I dont think the test proceeded before it was even launched)
Maybe you mean: "~0s"?